core/src/main/java/org/apache/iceberg/actions/SortStrategy.java

```diff
@@ -108,22 +84,12 @@ public RewriteStrategy options(Map<String, String> options) {
 
   @Override
```

Comment:
This override is not needed if we are only calling the super method

Reply:
Yeah, not required really. I have removed it.